proto/pulumi/provider.proto

```diff
@@ -112,11 +127,27 @@ service ResourceProvider {
     rpc GetMappings(GetMappingsRequest) returns (GetMappingsResponse) {}
 }
 
+// `ParameterizeRequest` is the type of requests sent as part of a [](pulumirpc.ResourceProvider.Parameterize) call. A
+// `ParameterizeRequest` may contain either:
+//
+// * a string array (`ParametersArgs`), intended to represent a set of command-line arguments so as to support
+//   instantiating a parameterized provider from a command-line invocation (e.g. to generate an SDK).
+// * a byte array accompanied by a name and version (`ParametersValue`), intended to represent a parameter embedded in a
+//   previously generated SDK.
```

Comment:
A previously generated sdk? I'm using this now directly when doing pulumi convert from terraform and embedding this in the "package" block.  Perhaps we can be more general or call out a couple examples?

Reply:
I will add a note to flesh this out -- my intention here was to signal that this is the case where a provider has probably been instantiated with a string list parameter and then produced a byte array parameter for later use (as we do in SDK generation). But that needs a better explanation and some examples as you say.